pandas/io/feather_format.py

```diff
@@ -128,7 +146,30 @@ def read_feather(
     with get_handle(
         path, "rb", storage_options=storage_options, is_text=False
     ) as handles:
-
-        return feather.read_feather(
-            handles.handle, columns=columns, use_threads=bool(use_threads)
-        )
+        if not use_nullable_dtypes:
+            return feather.read_feather(
+                handles.handle, columns=columns, use_threads=bool(use_threads)
+            )
+
+        dtype_backend = get_option("mode.dtype_backend")
+
+        if dtype_backend == "pandas":
+            from pandas.io._util import _arrow_dtype_mapping
+
+            pa_table = feather.read_table(
```

Comment:
Nit: Could you call `feather.read_table` once before the `if/elif`?

Reply:
done